src/LaravelJsonApi.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace CloudCreativity\LaravelJsonApi;
+
+class LaravelJsonApi
```

Comment:
Setting the default API now makes more sense on this class. Move it here and then deprecate the existing method.